src/libraries/testPackages/frameworkSettings/netstandard/settings.targets

```diff
@@ -11,6 +11,9 @@
 
     <!-- windows.winmd is not part of the framework -->
     <IgnoredReference Include="Windows" />
+
+    <!-- intentional dangling ref in System.Configuration.ConfigurationManager and System.DirectoryServices -->
+    <IgnoredReference Include="System.Security.Permissions" />
```

Comment:
Please move this to a package specific folder so that it only applies to packages we wish to have a dangling reference:
https://github.com/dotnet/runtime/tree/main/src/libraries/testPackages/packageSettings

I think you could use a path like
`testPackages/System.DirectoryServices/netstandard/settings.targets`
Have a look at https://github.com/dotnet/runtime/blob/2201016c1e13bdb9abf49e2e38cadf4ee0568df2/src/libraries/testPackages/build/packageTest.targets#L20-L25 for the conventions used to locate these suppressions.

Reply:
Done. Thanks